driver/razerkbd_driver.c

```diff
@@ -2414,11 +2556,16 @@ MODULE_DEVICE_TABLE(hid, razer_devices);
 static struct hid_driver razer_kbd_driver = {
     .name = "razerkbd",
     .id_table = razer_devices,
+
+    .input_mapping = razer_kbd_input_mapping,
+
     .probe = razer_kbd_probe,
     .remove = razer_kbd_disconnect,
 
     .event = razer_event,
-    .raw_event = razer_raw_event,
+    .raw_event = razer_raw_event
```

Comment:
leave the comma here

Reply:
Dear **@z3ntu**, I have not this comma at c4a29df (blackwiNdow fix), may be in some transitional commit